eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -33,6 +33,9 @@ parameters:
     default: '600s'
   - name: OSName
     type: string
+  - name: UseFederatedAuth
```

Comment:
Can you also plumb through the `EnvVars` variable here and into `deploy-test-resources.yml`, since identity may be using the same approach as net involving `PreSteps` and the `test-resources-pre.ps1` script to do additional resource setup.